### PR TITLE
Fixed rpm versioning

### PR DESCRIPTION
**Pull request by KjellKod**
This code does not have unit tests so I am not adding code coverage to it as of yet

### CODE REVIEW

**Comment by shwetabhandare** [comment]
:+1:

File: scripts/codeCoverage.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+/usr/local/probe/bin/cmake -DUSE_LR_DEBUG=ON -DVERSION=$VERSION -DCMAKE_CXX_COMPILER_ARG1:STRING=' -Wall -Werror -g -gdwarf-2 -fprofile-arcs -ftest-coverage -O0 -fPIC -m64 -Wl,-rpath -Wl,. -Wl,-rpath -Wl,/usr/local/probe/lib -Wl,-rpath -Wl,/usr/local/probe/lib64 ' -DCMAKE_CXX_COMPILER=/usr/local/probe/bin/g++ ..
+
+
+make -j
+./UnitTestRunner 
+cd ..
+
+
+
+PROJECT="DeathKnell"
```
Comment (john-gress) [inline scripts/codeCoverage.sh]: Clean up DeathKnell

Comment (KjellKod) [inline scripts/codeCoverage.sh]: will do

**Comment by john-gress** [comment]
:+1:

**Pull request merged**